Quaver.Shared/Database/Playlists/PlaylistManager.cs

```diff
@@ -133,6 +141,16 @@ private static void CreateTables()
             }
         }
 
+        /// <summary>
+        ///     Convert list to concurrent dictionary
+        /// </summary>
+        public static ConcurrentDictionary<TKey, TValue> ToConcurrentDictionary<TKey, TValue>
+            (this IEnumerable<TValue> source, Func<TValue, TKey> valueSelector)
+        {
+            return new ConcurrentDictionary<TKey, TValue>
+                (source.ToDictionary(valueSelector));
+        }
```

Comment:
Move this extension method into its own helper class/file - `ConcurrentDictionaryExtensions.cs`

The signature should also be on one line instead of two.